src/cpu/aarch64/jit_aarch64_sve_512_conv_kernel.cpp

```diff
@@ -0,0 +1,4417 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+* Copyright 2020 FUJITSU LIMITED
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "common/c_types_map.hpp"
+#include "common/nstl.hpp"
+#include "common/type_helpers.hpp"
+#include "common/utils.hpp"
+
+#include "cpu/aarch64/cpu_barrier.hpp"
+
+#include "cpu/aarch64/jit_aarch64_sve_512_conv_kernel.hpp"
+#include "cpu/platform.hpp"
+
+#include "cpu/aarch64/jit_op_imm_check.hpp"
+
+#define GET_OFF(field) static_cast<int32_t>(offsetof(jit_conv_call_s, field))
+#define KNx_L2_EFFECTIVE_CAPACITY ((512 - 64) * 1024)
```

Comment:
nit: Is the above `define` relevant here [especially because the prefix `KNx` seems to be specific for `x64` ISA]

Reply:
Yes, I certainly don't need to write here. I'll delete it.

Reply:
Looks good, thanks!